backend/qemu.pm

```diff
@@ -346,6 +347,54 @@ sub deflate_balloon ($self) {
     $self->handle_qmp_command({execute => 'balloon', arguments => {value => $ram_bytes}}, fatal => 1);
 }
 
+sub save_storage ($self, $args) {
+    my $vars = \%bmwqemu::vars;
+    my $bdc = $self->{proc}->blockdev_conf;
+    my $fname = $args->{filename};
+    my $rsp = $self->handle_qmp_command({execute => 'query-status'}, fatal => 1);
+    bmwqemu::diag("Saving storage devices (Current VM state is $rsp->{return}->{status})");
+    my $was_running = $rsp->{return}->{status} eq 'running';
+    if ($was_running) {
+        $self->inflate_balloon();
+        $self->freeze_vm();
+    }
+    mkpath("assets_public");
+    $bdc->for_each_drive(sub ($drive) {
+            my $size = $drive->{drive}->{size};
+            my $id = $drive->{id} . "-backup-$fname";
+            my $node = $drive->{drive}->{node_name};
+            my $my_node = "$node-$fname";
+            my $bck_file = "assets_public/$my_node-$vars->{NAME}.qcow2";
```

Comment:
Since this triggers an error `04-check_vars_docu.t` maybe we have to add it to `%var_blocklist`